test/run-drun/ok/scope-example-implicit.tc.ok

```diff
@@ -2,3 +2,8 @@ scope-example-implicit.mo:10.12-10.19: type error [M0096], expression of type
   async<$@anon-async-8.12> Int
 cannot produce expected type
   async<$f> Int
+because: Incompatible async scopes: 
+  $@anon-async-8.12
+ does not match 
+  $f
+ in top level
```

Comment:
Maybe the "in top level" are a bit redundant.
Maybe "in the expected type" would be better?